cc_library/example_scripts_js_scc/display-angular/src/app/display.ts

```diff
@@ -1,7 +1,10 @@
-const Device = require("sciler"); // production
+const Device = require("../../../../js_scc/index.js"); // production
+// const Device = require('sciler'); // production
```

Comment:
line 1 should be commented or just removed
line 2 should be uncommented

Reply:
will do that on npm upload